packages/react-internal/src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -139,22 +149,20 @@ describe('SearchBox', () => {
     expect(wrapper.find('input').prop('value')).toBe('test');
   });
 
-  it('handles setting null value', () => {
```

Comment:
Remind me why we decided to remove this? It was actually added in response to an issue #7449--I'm not sure now that adding explicit null handling was the right thing to do, but we should at least keep a test to confirm that there's not an exception on either initially setting or updating the value to null.

Reply:
Good catch! It looks this feature was not correctly ported with useControlableValue. I re-added the test and updated the base file so that value is set to a String. If the user passes in null for value they will receive “null”.

Reply:
I think the desired behavior is that if they pass `null` they get an empty string.

Reply:
Technically this is true, but I am not sure if it is worth the extra line of code to support just null given that it is not a valid value. It is also should encourage the use of undefined or a blank string in the unlikely scenario that null is referenced.

Reply:
The problem is that changing the behavior now will likely result in the literal string "null" showing up in people's apps. (And if they've been passing that in they've already either disabled the type check by casting, or their app is in JS.)